.github/workflows/linters.yml

```diff
@@ -45,4 +45,3 @@ jobs:
           npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
           [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
       - name: Stylelint Report
-        run: npx stylelint "**/*.{css,scss}"
```

Comment:
- [ ] Kindly make sure there are no custom changes to the linter config as required for every Microverse project.